java/bulk-import/bulk-import-runner/src/main/java/sleeper/bulkimport/job/runner/SparkFileInfoRow.java

```diff
@@ -34,6 +36,7 @@ public static FileInfo createFileInfo(Row row) {
         return FileInfo.builder()
                 .filename(row.getAs(FILENAME_FIELD_NAME))
                 .jobId(null)
+                .lastStateStoreUpdateTime(Instant.now())
```

Comment:
This isn't right as the state store isn't updated at this point. We should update the methods in the state store that store the file infos to set the time to Instant.now() when they are being stored. That's why the addFile method on the DynamoDBFileInfoStore doesn't check that the update time is non-null.

Reply:
If we deleted that line, the S3 state store would not be able to store the FileInfo as the field would be null. An alternative is to generate the update time in the state store and set it then, which we can do in #1381.

Reply:
OK, let's do it in that issue.